### ISSUE TITLE
Other A11y: Sign in: The language selection dropdown has no visible label

### ISSUE THREAD

**Issue by elisenwazuoke** [Expensify/App#77248]
## Action Performed:
Prerequisites:
- the user is not signed in

1. Open the Expensify app
2. Note how the language selector is displayed in the footer area.

## Expected Result:
The control should have a persistently visible label, e.g.:
- 'select language'

## Actual Result:
The dropdown button has no visible label provided.

## Area issue was found in:
Where did we find this issue?

## Failed WCAG checkpoints
WCAG 3.3.2 Labels or Instructions
https://www.w3.org/WAI/WCAG22/Understanding/labels-or-instructions

## User impact:
Providing clear and unambiguous labels and instructions (including examples of expected data formats) helps all users - but particularly those with cognitive, language, and learning disabilities - to enter information correctly and can prevent users from making incomplete or incorrect form submissions.

## Suggested resolution:
Add a permanently visible label to the control, e.g.:
- 'select language'
https://www.w3.org/WAI/WCAG22/Understanding/labels-or-instructions

## Environment(s)
Samsung Galaxy S24 Ultra - Android 14, Partner - Native (No mobile browser)
Apple iPhone 11 - iOS 17.5.1, H0T Mobile - Native (No mobile browser)

## Screenshots/Videos
![Image](https://github.com/user-attachments/assets/eb05642b-04ed-4c47-9c0e-1d3b9c4a94fc)

https://github.com/user-attachments/assets/ded397b5-e6fe-4bb3-9263-2717d8789295

**Version Number:** Production
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36628/community-issues/7004760
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by rushatgabhane** [comment]
fixed by https://github.com/Expensify/App/issues/79239

**Comment by rushatgabhane** [comment]
This issue was retested in **BFV cycle 544085 (Native Apps)** on 2026-04-28 and is **Not Fixed**.

### Screenshots / Videos
- https://platform.applause.com/services/links/v1/external/866bea55eabfad4ea0ef79a3f387edbdcf3d0128957ef65716bb2e7e621ecc18

### Retest notes
Failed: 
The issue is not fixed

The issue is not fixed. A visible label is still not provided for the language selection dropdown. 

Issue verified on: 
iOS Version: 26.4.1
App version: 9.3.63
Please see the attached screenshot

**Comment by mallenexpensify** [comment]
### Payment Summary
Contributor+: @jayeshmangwani  due $250 via NewDot

**Comment by quinthar** [comment]
Approved $250 to @jayeshmangwani
